tests/Util/yaml_fixtures/array_double_quoted_foo_value.test

```diff
@@ -0,0 +1,14 @@
+caddy:
+    ports:
+      - "foo:foo"
+    volumes:
+      - caddy_data
+===
+$data['caddy']['ports'][] = 'bar:baz';
+===
+caddy:
+    ports:
+      - "foo:foo"
+      - 'bar:baz'
```

Comment:
This test passes. but not w/ `"80:80"`

Reply:
So we don't change the existing double quotes to single quotes. It's only that the NEW keys we add are double quotes? If so, I think it's fine.